src/dotnet-reqube/Program.cs

```diff
@@ -36,22 +36,24 @@ private static void Convert(Options options)
                 var sonarQubeReports = Map(report);
 
                 // We need to write dummy report because SonarQube MSBuild reads a report from the root
-                WriteReport(CombineOutputPath(options, options.Output), SonarQubeReport.Empty);
-
-                foreach (var sonarQubeReport in sonarQubeReports)
+                if (string.IsNullOrEmpty(options.Project))
                 {
-                    var projectDirectory = CombineOutputPath(options, sonarQubeReport.ProjectName);
-                    if (!Directory.Exists(projectDirectory))
+                    WriteReport(CombineOutputPath(options, options.Output), SonarQubeReport.Empty);
+
+                    foreach (var sonarQubeReport in sonarQubeReports)
                     {
-                        Directory.CreateDirectory(projectDirectory);
-                    }
+                        var filePath = CombineOutputPath(options, Path.Combine(sonarQubeReport.ProjectName, options.Output));
 
-                    var filePath = Path.Combine(projectDirectory, options.Output);
+                        WriteReport(filePath, sonarQubeReport);
+                    }
 
-                    WriteReport(filePath, sonarQubeReport);
+                    TryWriteMissingReports(report.Information.Solution, options, sonarQubeReports);
+                }
+                else
+                {
+                    var projectToWrite = sonarQubeReports.FirstOrDefault(r => string.Equals(r.ProjectName, options.Project, StringComparison.OrdinalIgnoreCase));
```

Comment:
Probably need to add a log here if the project is not found.

Reply:
I added a log output, however the project won't be found also in case the resharper xml contains no issues for the project, so I just added it as info (i.e. not to Error stream) and write the report anyway.